spanner/go.mod

```diff
@@ -31,3 +31,6 @@ require (
 	golang.org/x/text v0.3.7 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 )
+
+// To prevent failing builds as proto changes are not available in go-genproto until Public GA
+replace google.golang.org/genproto v0.0.0-20221014173430-6e2ab493f96b => github.com/harshachinta/go-genproto v0.0.0-20221020104338-f731337b715d
```

Comment:
- Please check if we can create branch on the [googleapis/google-cloud-go](https://github.com/googleapis/google-cloud-go) and do the Proto schema changes. 
- If we are able to create, use branch from [googleapis/google-cloud-go](https://github.com/googleapis/google-cloud-go) here.

Reply:
In addition to `go.mod` file, the `go-genproto` dependency needs to be updated in `internal/kokoro/build_samples.sh` file as well for the kokoro CI build to pass successfully.  This is because the `build_samples.sh` file builds the `golang-samples` file which has `go-genproto` pointing to the release version, and thus failing the build.

But the `build_samples.sh` file change is not needed for our proto column feature and hence reverted those changes.